devdox/run_migrations.py

```
@@ -448,6 +636,25 @@ async def run_ultimate_migrations():
     print("✅ Adapts to future schema changes")
     print("✅ Works in any environment")
+    print("settings.MIGRATIONS_PATH ", settings.MIGRATIONS_PATH)
+    # Verify migrations directory exists
+    if not os.path.exists(settings.MIGRATIONS_PATH):
+        print(f"❌ Migrations path does not exist: {settings.MIGRATIONS_PATH}")
+        print("   Ensure Render disk is mounted at /data")
+        return False
+
+    try:
+        ensure_migrations_directory()
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)